Temperatures_kjb589/kjb589_Temperatures.py

```diff
@@ -9,20 +9,25 @@
 # considered closest to zero (for instance, if the temperatures are -5 and 5,
 # then display 5).
 #
+# <mark >BAB:</mark> Good explanation I can't find anything to critize here.
```

Comment:
Remove review comments; instead, integrate them into your code or explain why you didn't in a PR.

Reply:
Integrated variable name changes, edited some of my own comments to fix grammar, and removed peer review comments.